concent_api/core/views.py

```diff
@@ -214,14 +280,23 @@ def handle_send_reject_report_computed_task(client_message):
             client_message.TYPE,
             force_report_computed_task.task_to_compute.compute_task_def['task_id'],
             client_message.serialize(),
-            status = ReceiveStatus
+            provider_public_key  = force_report_computed_task_from_database.last().auth.provider_public_key_bytes,
+            requestor_public_key = client_public_key,
+            status               = ReceiveStatus
         )
-
         return HttpResponse("", status = 202)
 
     if current_time <= force_report_computed_task.task_to_compute.compute_task_def['deadline'] + settings.CONCENT_MESSAGING_TIME:
-        ack_message             = Message.objects.filter(task_id = client_message.cannot_compute_task.task_to_compute.compute_task_def['task_id'], type = message.AckReportComputedTask.TYPE)
-        previous_reject_message = Message.objects.filter(task_id = client_message.cannot_compute_task.task_to_compute.compute_task_def['task_id'], type = message.RejectReportComputedTask.TYPE)
```

Comment:
This change does not make sense on its own. It should be in the same commit that adds the lines below.

Reply:
:+1: